packages/wasi-common/package.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "@bytecodealliance/wasi-common",
```

Sounds good, but perhaps let's make it release-specific and note that this is not a "proper" implementation but a "shim" -

```suggestion
    "name": "@bytecodealliance/preview2-shim",
```